src/payment/CardHolderInformation.jsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Field } from 'redux-form';
+import { injectIntl, intlShape, FormattedMessage } from '@edx/frontend-i18n';
+
+import { countryOptionsSelector } from './data/selectors';
+import messages from './CardHolderInformation.messages';
+
+class CardHolderInformation extends React.Component {
+  constructor(props) {
+    super(props);
+    this.countryOptions = [{
+      value: '',
+      label: props.intl.formatMessage(messages['payment.card.holder.information.country.options.empty']),
+    }].concat(props.countryOptions);
+  }
+
+  renderCountryOptions() {
+    const items = [];
+    for (let i = 0; i < this.countryOptions.length; i += 1) {
+      const { value, label } = this.countryOptions[i];
+      items.push(<option key={value} value={value}>{label}</option>);
+    }
+    return items;
+  }
+
+  render() {
+    return (
+      <div className="container-fluid py-5">
+        <h2 className="h6">
+          <FormattedMessage
+            id="payment.card.holder.information.heading"
+            defaultMessage="Card Holder Information"
+            description="The heading for the credit card holder information form"
+          />
+        </h2>
+
+        <div className="row">
+          <div className="col-lg-6 form-group">
+            <label htmlFor="firstName">
+              <FormattedMessage
```

Comment:
I believe using <FormattedMessage> will cause it to insert an unncecessary <span> element in here.  (I _think_ - worth double checking me).  We might want to use `this.props.intl.formatMessage()` instead to avoid the unnecessary DOM element.

Reply:
Is there a negative consequence of the unnecessary DOM element?